Frontend/src/packages/dashboard/components/FileRenderer/FileContainer.tsx

```diff
@@ -38,6 +38,7 @@ const IconContainer = styled.div<styledProps>`
 
 function FileContainer({ name, id, selectedFile, setSelectedFile }: Props) {
   const handleClick = () => {
+    console.log(id)
```

Comment:
remember to remove debugging statements if possible, its a good habit to start doing